NumLib/ODESolver/TimeLoopSingleODE.h

```diff
@@ -10,6 +10,8 @@
 #pragma once
 
 #include "NumLib/DOF/GlobalMatrixProviders.h"
+#include "ProcessLib/StaggeredCouplingTerm.h"
+
```

Comment:
I'm not sure if this can be fwd. declared. Maybe it would be better to move StaggeredCouplingTerm into NumLib?